### ISSUE TITLE
Chat only mode (new feature suggestion)

### ISSUE THREAD

**Issue by zkent** [FabricLabs/soundtrack#90]
Often I want to keep chat open and turn off music or play something different.  But muting and hiding video don't save any system resources (CPU over 50% on my old system).  It would be nice if there were a chat only mode where no video or audio is involved.

**Comment by zbeekman** [comment]
It seems like this thread might profit from being renamed, as this issue is more about resource usage, than “chat only” since `/stream off` as well as muting and making the chat window bigger accomplish the goal of ‘chat only’ just my $0.02

**Comment by zkent** [comment]
I agree.  I can't even run the site on my currently PC anymore if I am
working. My biggest issue here is CPU usage.

On Thu, Feb 5, 2015 at 9:59 AM, Izaak Beekman <notifications@github.com>
wrote:

> It seems like this thread might profit from being renamed, as this issue
> is more about resource usage, than “chat only” since /stream off as well
> as muting and making the chat window bigger accomplish the goal of ‘chat
> only’ just my $0.02
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/martindale/soundtrack.io/issues/90#issuecomment-73059693>
> .
>



-- 
*Zachary Kent*
Web Solutions Architect
View my LinkedIn profile <http://www.linkedin.com/in/zacharykent>

**Comment by martindale** [comment]
Let's figure out how to tackle the CPU use problem; the API exists for anyone to develop a chat-only mode, but it won't be a priority for me for a while.  Feel free to submit a pull request that enables it, though!

I've opened #137 to handle issues / proposals for solving the CPU issue.
